server/src/resources/event.rs

```diff
@@ -5,7 +5,7 @@ use westiny_common::components::Input;
 #[derive(Debug, Eq, PartialEq)]
 pub enum ClientNetworkEvent {
     ClientConnected(ClientID),
-    ClientDisconnected(ClientID),
+    ClientDisconnected(ClientID, String),
```

Comment:
please use named fields here, because it's not clear what this string stands for.
(I've found out, it's player name, but I had to see it's usages to find out)

Reply:
I have added a PlayerName strong type (similarly to ClientId) to denote it